website/templates/website/volunteer_list.html

```diff
@@ -0,0 +1,49 @@
+{% extends "website/default.html" %}
+
+{% block title %} {{ block.super }} -- Volunteer List {% endblock %}
+
+{% block content %}
+  {{ block.super }}
+  <div class="container">
+    <div class="event-list-header">
+      <h1 class="event-list-title">Volunteer List</h1>
+    </div>
+    <div class="event-detail-card">
+      <div class="event-card-container">
+          <table class="table">
+            <caption style="caption-side: top"> {{ event.name }} </caption>
+              <thead class="thead-dark">
+                <tr>
+                  <th scope="col"> Volunteer </th>
+                  <th scope="col"> Action </th>
+                </tr>
+              </thead>
+          {% if event %}
+            {% for Volunteer in assign_list %}
```

Comment:
lower case variable name
```suggestion
            {% for volunteer in assign_list %}
```